diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/StringUtilsTest.kt

```diff
@@ -0,0 +1,38 @@
+package org.cqfn.diktat.ruleset.utils
+
+import com.pinterest.ktlint.core.ast.ElementType.CALL_EXPRESSION
+import com.pinterest.ktlint.core.ast.ElementType.FUN
+import com.pinterest.ktlint.core.ast.ElementType.PROPERTY
+import com.pinterest.ktlint.core.ast.ElementType.WHITE_SPACE
+import org.junit.Assert
+import org.junit.jupiter.api.Test
+
+class StringUtilsTest {
```

Comment:
Please provide some examples with parsing of multiline code fragments, invalid code fragments.